projects/components/src/lib/wizard-progress-tracker/wizard-progress-tracker.component.ts

```diff
@@ -0,0 +1,28 @@
+import { CommonModule } from '@angular/common';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { SubSink } from 'subsink';
+import { WizardStepLink } from '../models/wizard-step-link';
+import { SvgIconComponent } from '../svg-icon/svg-icon.component';
+
+@Component({
+  selector: 'mz-wizard-progress-tracker',
+  templateUrl: './wizard-progress-tracker.component.html',
+  styleUrls: ['./wizard-progress-tracker.component.scss'],
+  standalone:  true,
+  imports: [CommonModule, SvgIconComponent]
+})
+export class WizardProgressTrackerComponent {
```

Comment:
Can we rename this to ProgressTrackerComponent please? Just trying to shorten it and less specific to wizard